Source/EmbeddedBoundary/ParticleScraper.H

```diff
@@ -153,20 +153,17 @@ scrapeParticles (PC& pc, const amrex::Vector<const amrex::MultiFab*>& distance_t
 template <class PC, class F, std::enable_if_t<amrex::IsParticleContainer<PC>::value, int> foo = 0>
 void
 scrapeParticles (PC& pc, const amrex::Vector<const amrex::MultiFab*>& distance_to_eb,
-                 int lev_min, int lev_max, F&& f)
+                 int lev_min, int lev_max, F&& f,const amrex::Real dt)
 {
     BL_PROFILE("scrapeParticles");
 
     for (int lev = lev_min; lev <= lev_max; ++lev)
     {
         const auto plo = pc.Geom(lev).ProbLoArray();
         const auto dxi = pc.Geom(lev).InvCellSizeArray();
-#ifdef AMREX_USE_OMP
-#pragma omp parallel if (amrex::Gpu::notInLaunchRegion())
-#endif
```

Comment:
Could you explain why this was removed?

Reply:
It was maybe not up-to-date with https://github.com/ECP-WarpX/WarpX/pull/4418